hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_2_0/3664-shorten-first-prefetch.yaml

```
@@ -0,0 +1,4 @@
+---
+type: fix
+issue: 3664
+title: "The first JPA prefetch window was shortened to avoid slow first bundles."
```

Suggested slightly more descriptive changelog

```suggestion
title: "Initial page loading has been optimized to reduce the number of prefetched resources. This should improve the speed of initial search queries in many cases."
```